app/policies/note_policy.rb

```diff
@@ -9,7 +9,7 @@ def resolve
       if user.staff_level?
         scope.all
       else
-        user.properties.property_managers
+        scope.where(user: user)
```

Comment:
I don't remember if a property manager is allowed to edit/destroy any note as long as it was created by one of their co-managers.

If that is the case then this should be `scope.where(user: user.properties.property_managers)`

Reply:
I think this is fine for now, we can come back to it if we need to